core/controllers/admin.py

```diff
@@ -848,3 +848,16 @@ def post(self):
             self.render_json({})
         else:
             raise self.InvalidInputException('This app cannot send emails.')
+
+
+class InteractionsByExplorationId(base.BaseHandler):
+    """Handler to show a list of interaction id's for a exploration id."""
+
+    @acl_decorators.can_access_admin_page
+    def get(self):
+        exploration_id = self.request.get('exploration_id')
+        exploration = exp_fetchers.get_exploration_by_id(exploration_id)
+        interaction_ids = {
+            state.interaction.id for state in exploration.states.items()
```

Comment:
I was thinking of creating (reusable) `get_used_interactions_list` (or something around the same [not sure about the naming]) inside the exploration class and call `exploration.get_used_interactions_list` here just to avoid exposing the internal properties of the object in the controller layer. I'm also fine with the current structure as this handler is for the admin page.